scipy/stats/stats.py

```diff
@@ -5718,3 +5720,175 @@ def rankdata(a, method='average'):
 
     # average method
     return .5 * (count[dense] + count[dense - 1] + 1)
+
+
+def ratio_unif(f, size=1, x1=None, x2=None, y2=None, shift=0, maxiter=None,
+               bounds=None):
+    """
+    Generate random variables using ratio-of-uniforms method.
+
+    Given a continuous univariate density f and a constant c, define the set
+    A(c) := {(x, y) : 0 < y <= sqrt(f(x/y + c)) }. If (U, V) is a random
```

Comment:
This mathematical notation won't be easy to understand by a lot of people. If it's not necessary to know this to use the function then I would put it in the notes.

Reply:
i tried to simplify it, but the definition of the set A(c) is crucial for understanding the algorithm